assets/konnected.css

```diff
@@ -1,3 +1,12 @@
+@font-face {
+  font-family: Inter;
+  src: url('https://cdn.shopify.com/s/files/1/2085/3145/files/Inter-Medium.woff2') format('woff2'),
+      url('https://cdn.shopify.com/s/files/1/2085/3145/files/Inter-Medium.woff') format('woff');
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+}
+
```

Comment:
why do we need to add it again? Isn't this already used for the `body-font` css variable?

Reply:
@heythisisnate can't use 500 font weight of Inter shopify, only regular declared
![image](https://github.com/konnected-io/shopify-dawn/assets/32586764/84c78fb6-f4a6-4e2b-8818-4aa7b57c66e3)

Reply:
We're loading Inter regular twice here in the Shopify theme. Just change the Heading font to Inter Medium and then Shopify will automatically import it. This avoids one extra asset fetch on page load.
![Screenshot 2023-10-06 at 8 46 36 AM](https://github.com/konnected-io/shopify-dawn/assets/12016/a9043ed2-73ed-4aae-9ad3-8dab9e3db46f)

Reply:
ok ok tnx